=== tests/cases/compiler/funduleUsedAcrossFileBoundary_file1.ts ===
declare function Q<T>(value: T): string;
>Q : typeof Q
>T : T
>value : T
>T : T

declare module Q {
>Q : typeof Q

    interface Promise<T> {
>Promise : Promise<T>
>T : T

        foo: string;
>foo : string
>类型别名 :=> 文字
    }
    export function defer<T>(): string;
>defer : <T>() => string
>T : T
}

=== tests/cases/compiler/funduleUsedAcrossFileBoundary_file2.ts ===
function promiseWithCancellation<T>(promise: Q.Promise<T>) {
>promiseWithCancellation : <T>(promise: Q.Promise<T>) => void
>T : T
>promise : Q.Promise<T>
>Q : any
>类型别名 :=> 任意
>Promise : Q.Promise<T>
>T : T

    var deferred = Q.defer<T>(); // used to be an error
>deferred : string
>类型别名 :=> 文字
>Q.defer<T>() : string
>类型别名 :=> 文字
>Q.defer : <T>() => string
>Q : typeof Q
>defer : <T>() => string
>T : T
}
